packages/client/src/LensClient.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { Wallet } from 'ethers';
+
+import LensClient, { Credentials, mumbai, ReactionTypes } from '.';
+
+let credentials: Credentials;
+const getCredentials = async (wallet: Wallet) => {
+  if (credentials) {
+    return credentials;
+  }
+  const client = LensClient.init(mumbai);
```

Comment:
Am I correct thinking this test suite hits the real API?

Part of me like the idea of testing against the real schema... but maybe we should consider to limit these to run on CI as part of a smoke tests suite? Thoughts?

Reply:
there is a ticket to "Run only affected packages lint/tests on CI" https://avara.height.app/inbox/T-5453 and initially there are hardly any tests here so I'd worry about that later